Sources/SwiftDocC/Utility/FoundationExtensions/URL+IsAbsoluteWebURL.swift

```diff
@@ -0,0 +1,20 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021 Apple Inc. and the Swift project authors
```

Comment:
```suggestion
 Copyright (c) 2022 Apple Inc. and the Swift project authors
```
(since these are new files)

Reply:
Got it